draft-ietf-quic-recovery.md

```diff
@@ -496,9 +500,7 @@ be considered an RTT sample.
 
 Until the server has validated the client's address on the path, the amount of
 data it can send is limited, as specified in Section 8.1 of {{QUIC-TRANSPORT}}.
-Data at Initial encryption MUST be retransmitted before Handshake data and
-data at Handshake encryption MUST be retransmitted before any ApplicationData
-data.  If no data can be sent, then the PTO alarm MUST NOT be armed until
+If no data can be sent, then the PTO alarm MUST NOT be armed until
 data has been received from the client.
```

Comment:
s/data/valid packets

"data", to me, implies payloads of CRYPTO or STREAM freams.

Reply:
Changed to datagram, PTAL